src/App.js

```diff
@@ -1,41 +1,12 @@
 import "./App.css";
 import logo from "./logo.png";
-
+import LandingPage  from "./pages/landing-page/landingPage.jsx";
```

Comment:
instead of doing this you can create an index.js file inside pages forder and export it from there and then while importing you would only have to write 
```
import LandingPage  from "./pages";
```

Reply:
This seems cleaner will update it